lib/suspenders/generators/stylesheet_base_generator.rb

```diff
@@ -0,0 +1,23 @@
+require "rails/generators"
+
+module Suspenders
+  class StylesheetBaseGenerator < Rails::Generators::Base
+    source_root File.expand_path(
+      File.join("..", "..", "..", "templates"),
+      File.dirname(__FILE__))
+
+    def add_stylesheet_gems
+      gem "bourbon", "5.0.0.beta.5"
+      gem "neat", "~> 1.7.0"
+      gem "refills", group: [:development, :test]
+    end
+
+    def add_css_config
+      copy_file(
+        "application.scss",
+        "app/assets/stylesheets/application.scss",
+        force: true,
+      )
```

Comment:
I mean: leave this line as-is, but add an additional check for `app/assets/application.css` and if it exists print a warning.

I doubt this will come up, but it's considerate to give them the heads-up.